### ISSUE TITLE
Layer Normalization

### ISSUE THREAD

**Issue by enpasos** [deepjavalibrary/djl#1057]
## Description

I would like to use Layer Normalization just the way I use [Batch Normalization in DJL](https://javadoc.io/doc/ai.djl/api/latest/ai/djl/nn/norm/BatchNorm.html).
What would be the best way to get there?

## Layer Norm References
- [paper](https://arxiv.org/abs/1607.06450)
- [api pytorch](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html)
- [api mxnet](https://mxnet.apache.org/versions/1.8.0/api/python/docs/api/gluon/nn/index.html#mxnet.gluon.nn.LayerNorm)
- [api tensorflow](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization)

**Comment by zachgk** [comment]
Right now, DJL doesn't have an API to access the LayerNorm. But, the underlying engine you are using should have it implemented. If you call the engine directly, you can still access their implementation.

You can use BatchNorm as an example of what that looks like or find some more self contained examples in the engine's implementation of the NDArray interface. Which engine are you using?

**Comment by enpasos** [comment]
Right now I am using PyTorch.
I will follow your advice: Use BatchNorm as an example.
Would it be an option to integrate it into DJL if it comes as a pull request?

**Comment by zachgk** [comment]
> Would it be an option to integrate it into DJL if it comes as a pull request?

Absolutely!
